.gitmodules

```diff
@@ -20,7 +20,7 @@
 	shallow = true
 [submodule "library/stdarch"]
 	path = library/stdarch
-	url = https://github.com/rust-lang/stdarch.git
+	url = https://github.com/usamoi/stdarch.git
 	shallow = true
```

Comment:
Hm, you may need to additionally specify your branch?

Reply:
The data in .gitmodules doesn't matter, what you need to do is `cd library/stdarch` and check out your branch in that directory.